index.html

```diff
@@ -2211,13 +2211,15 @@ <h2>
 To enable the full functionality of <a>DIDs</a> and <a>DID documents</a> on a
 particular <a>DID registry</a>, a <a>DID method</a> specification MUST specify
 how each of the following <abbr title="Create, Read, Update, Delete">CRUD</abbr>
-operations is performed by a client. Specify each operation to the level of
-detail necessary to build and test interoperable client implementations with the
-<a>DID registry</a>. These operations can effectively be used to perform all the
-operations required of a CKMS (cryptographic key management system). For
-example, key registration, key replacement, key rotation, key recovery, and key
+operations is performed by a client. In the event that an operation is 
+unsupported by a <a>DID method</a>, the <a>DID method</a> specification 
+MUST specify that it's not supported. If an operation is supported, it 
```

Comment:
```suggestion
MUST specify that it is not supported. If an operation is supported, it 
```

Reply:
Avoid use of contractions in spec language as contractions tend to be more difficult to read for non-native speakers and makes normative statements feel like informal dialog.